ui/helpers/utils/util.js

```diff
@@ -209,24 +209,29 @@ export function getRandomFileName() {
 }
 
 /**
- * Shortens an Ethereum address for display, preserving the beginning and end.
- * Returns the given address if it is no longer than 10 characters.
- * Shortened addresses are 13 characters long.
+ * Shortens the given string, preserving the beginning and end.
+ * Returns the string it is no longer than truncatedCharLimit.
  *
- * Example output: 0xabcde...12345
- *
- * @param {string} address - The address to shorten.
- * @returns {string} The shortened address, or the original if it was no longer
- * than 10 characters.
+ * @param {string} stringToShorten - The string to shorten.
+ * @param {number} truncatedCharLimit - The maximum length of the string.
+ * @param {number} truncatedStartChars - The number of characters to preserve at the beginning.
+ * @param {number} truncatedEndChars - The number of characters to preserve at the end.
+ * @returns {string} The shortened string.
  */
-export function shortenAddress(address = '') {
-  if (address.length < TRUNCATED_NAME_CHAR_LIMIT) {
-    return address;
+export function shortenAddress(
+  stringToShorten = '',
+  truncatedCharLimit = TRUNCATED_NAME_CHAR_LIMIT,
```

Comment:
Given there is many positional arguments with the same type, would it be more usable if we use an object for the optional properties (other than the string itself)?

Reply:
Right, I was tryign to simplify the first working version but missed this one removed in
https://github.com/MetaMask/metamask-extension/pull/25252/commits/7ae1d604fc3c9c559b2c4f32b62ad0e085184ea1